FWCore/Integration/test/run_TestEDAlias.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+test=testEDAlias
```

Comment:
How is this test run? I didn't notice anything in BuildFile.xml.

Reply:
Sigh, I forgot to add it again to BuildFile.xml, thanks for catching.